tests/test_cli.py

```
@@ -0,0 +1,834 @@
+            cli.main()
+
+        assert e.value.code == 0
+        mock_run.assert_called_once()
+        mock_exit.assert_called_with(0)
+
+    @patch("os.path.exists", return_value=True)
+    @patch("sys.argv", ["biblebot", "auth", "logout"])
+    @patch("biblebot.auth.interactive_logout")
+    @patch("biblebot.cli.asyncio.run")
+    @patch("sys.exit")
+    def test_auth_logout_command(self, mock_exit, mock_run, mock_logout, mock_exists):
```

Ruff (0.12.2)

Unused method argument: `mock_exists`

(ARG002)